plugins/pixart-rf/fu-pxi-ble-device.c

```diff
@@ -714,11 +735,11 @@ fu_pxi_ble_device_write_firmware(FuDevice *device,
 	/* progress */
 	fu_progress_set_id(progress, G_STRLOC);
 	fu_progress_add_flag(progress, FU_PROGRESS_FLAG_GUESSED);
```

Comment:
remove GUESSED if these are actually measured now.

Reply:
Revised in b623d0cd5cad59a43bc7c4016fe867bb8013caf6.